services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/device/AutoProvisioner.java

```diff
@@ -69,7 +66,7 @@
 
     private TenantInformationService tenantInformationService = new NoopTenantInformationService();
 
-    private DownstreamSenderFactory downstreamSenderFactory;
+    private ProtonBasedDownstreamSender protonBasedDownstreamSender;
```

Comment:
FMPOV this should be of type `org.eclipse.hono.adapter.client.telemetry.EventSender`